src/vs/workbench/services/textfile/common/textFileEditorModel.ts

```diff
@@ -698,7 +698,7 @@ export class TextFileEditorModel extends BaseTextEditorModel implements ITextFil
 			// Save to Disk
 			// mark the save operation as currently pending with the versionId (it might have changed from a save participant triggering)
 			this.logService.trace(`doSave(${versionId}) - before updateContent()`, this.resource);
-			return this.saveSequentializer.setPending(newVersionId, this.fileService.updateContent(this.lastResolvedDiskStat.resource, this.createSnapshot(), {
+			return this.saveSequentializer.setPending(newVersionId, this.fileService.updateContent(this.lastResolvedDiskStat.resource, this.createSnapshot()!, {
```

Comment:
@mjbvz please avoid "`!`"

Reply:
I'm not sure how to fix this one properly. We really don't want `updateContent` to take `undefined` as an argument. I think throwing an error if createSnapshot returns null seems reasonable as I believe the code would crash anyways